### DISCUSSION TITLE
Search only in specific folders and handle .lnk files

### DISCUSSION THREAD

**Discussion by Netkuszkusz** [Flow-Launcher/Flow.Launcher#2309] [Ideas]
Hey! I'm pretty new in Flow Launcher, just came from the old Lanchy. I love the new features, and I left almost every plugins switched on except 'Explorer', 'Program' and 'Browser Bookmarks'. Now it's clean, but I miss my own links. I really don't like the 'Explorer' plugin, it's confusing and not seems very adjustable.

So, can you suggest a plugin or method to use my curated shortcuts folder in Flow Launcher? The folder contains .lnk files for applications, folders and files. I did this with Launchy, and I miss this usage from Flow. I don't need mass search, because I use Everything separately, and don't need program indexing from start menu and such, because I want to see ONLY what I choosed.

It would be nice if the launcher could hide the .lnk extension when I write something to the launcher and of course I want to see the icon from the shortcut. Thanks!

**Comment by deefrawley** [comment]
Just bringing the conversation across from Reddit - https://www.reddit.com/r/FlowLauncher/comments/16190vq/search_only_in_specific_folders_and_handle_lnk/

The Explorer plugin simply uses Windows Indexing or Everything to search and displays the results. If you are using Everything separately you could set up Windows Indexing to only index your curated folder and that should be all it will search. You can give it a keyword to get only those results. The other way is to make custom shortcuts to your links but you would need to manually set one up for each shortcut.
